Rubberduck.Parsing/Symbols/DeclarationFinder.cs

```diff
@@ -455,9 +440,8 @@ private IEnumerable<Declaration> FindEvents(Declaration module)
             Debug.Assert(module != null);
 
             var members = Members(module.QualifiedName.QualifiedModuleName);
-            return members == null 
-                ? Enumerable.Empty<Declaration>() 
-                : members.Where(declaration => declaration.DeclarationType == DeclarationType.Event);
+            return members?.Where(declaration => declaration.DeclarationType == DeclarationType.Event)
```

Comment:
it would be useful to investigate why `members` is considered nullable here in the first place. `null` `IEnumerable`s are a smell in and of themselves...